File: libraries/hdm/functions/types/request/AutonomousDUStateChangeDTO.java
package hdm.functions.types.request;

import hdm.functions.types.response.ResponseDTO;
import hdm.functions.types.struct.AutonOpResultStructDTO;
import java.io.Serializable;

public class AutonomousDUStateChangeDTO
  implements ResponseDTO, Serializable
{
  private AutonOpResultStructDTO[] autonomousOperationResult;

  public AutonOpResultStructDTO[] getAutonomousOperationResult()
  {
    return this.autonomousOperationResult;
  }

  public void setAutonomousOperationResult(AutonOpResultStructDTO[] autonomousOperationResult)
  {
    this.autonomousOperationResult = autonomousOperationResult;
  }
}

File: libraries/hdm/functions/types/request/AutonomousDUStateChangeDTO.js
/* @org.eclipse.vjet.dsf.resource.utils.CodeGen("VjoGenerator") */
vjo.ctype('hdm.functions.types.request.AutonomousDUStateChangeDTO') //< public
//> needs hdm.functions.types.struct.AutonOpResultStructDTO
.satisfies('hdm.functions.types.response.ResponseDTO')
.satisfies('java.io.Serializable')
.protos({
    autonomousOperationResult:null, //< private AutonOpResultStructDTO[]
    //> public AutonOpResultStructDTO[] getAutonomousOperationResult()
    getAutonomousOperationResult:function(){
        return this.autonomousOperationResult;
    },
    //> public void setAutonomousOperationResult(AutonOpResultStructDTO[] autonomousOperationResult)
    setAutonomousOperationResult:function(autonomousOperationResult){
        this.autonomousOperationResult=autonomousOperationResult;
    }
})
.endType();